operations/vouchers.md

```diff
@@ -146,21 +141,275 @@ Returns all rate vouchers filtered by [Service](services.md#service), voucher co
+
+Updates information about the specified vouchers.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/vouchers/update`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+    "VoucherUpdates": [
+        {
+            "VoucherId": "fe568bbd-1ecb-4bb2-bf77-96c3698de20d",
+            "Name": { "Value": "Weekend Voucher" },
+            "Type": { "Value": "Public" },
+            "CompanyId": null,
+            "AssignedRateIds": { "Value": 
+                [
+                    "181f8cdd-04ee-4bf5-ba3e-44c108eca3cb",
+                    "8bebeddc-9137-432d-810c-1b998a90ac9a"
+                ]
+            },
+            "OccupiableIntervalStartUtc": null,
+            "OccupiableIntervalEndUtc": null,
+            "ExternalIdentifier": null
+        },
+        {
+            "VoucherId": "f4a9942c-2616-4074-b1f4-4b959515e933",
+            "Name": { "Value": "Weekend Voucher" },
+            "Type": { "Value": "PartnerCompany" },
+            "CompanyId": { "Value": "3506994b-3c0b-49ba-9f57-ac4700641440" },
+            "AssignedRateIds": { "Value": 
+                [
+                    "181f8cdd-04ee-4bf5-ba3e-44c108eca3cb",
+                    "8bebeddc-9137-432d-810c-1b998a90ac9a"
+                ]
+            },      
+            "OccupiableIntervalStartUtc": { "Value": "2023-12-31T22:00:00Z" },
+            "OccupiableIntervalEndUtc": { "Value": "2024-01-01T22:00:00Z" },
+            "ExternalIdentifier": { "Value": "VCHR-278" }
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](enterprises.md#enterprise). Required when using a [Portfolio Access Token](../guidelines/multi-property.md), ignored otherwise. |
+| `VoucherUpdates` | array of [Voucher update parameters](#vouchers-update-parameters) | required, max 1000 items | Vouchers to be added. |
+
+#### Vouchers update parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `VoucherId` | string | required | Unique identifier of the [Voucher](#voucher). |
+| `Name` | [String update value](_objects.md#string-update-value), max 128 characters | optional | Internal name of the voucher \(or `null` if the name should not be updated\). |
+| `Type` | [String update value](_objects.md#string-update-value) [Type](#voucher-type), max 128 characters | optional | type of the voucher \(or `null` if the name should not be updated\). |
```

Comment:
@MikeAdamsMews I'm not sure with this one, which is `enum` but `update value`, with limited length 😅 
`[String update value](_objects.md#string-update-value) [Type](#voucher-type), max 128 characters`

Reply:
Yes, I see the problem. My suggestion is to move `Type` into the description.